Sample_v3/Samples/SwiftUISimpleChat/ChatView.swift

```diff
@@ -7,38 +7,119 @@ import SwiftUI
 
 @available(iOS 13, *)
 struct ChatView: View {
-    @State var channelId: ChannelId
+    /// SwiftUI expects view initialization to be cheap, so we hold the a `ChannelController` builder to be called in `onAppear`.
+    let channelBuilder: () -> ChannelController.ObservableObject
```

Comment:
Let's wait for #488 to be merged. After that, you can remove this and just make `synchronize()` call in `onAppear`.

Reply:
cc @mantergo